www/front_src/src/Resources/Graph/Performance/Graph/index.tsx

```diff
@@ -490,6 +503,27 @@ const GraphContent = ({
 
   const commentTitle = isCommentPermitted ? '' : t(labelActionNotPermitted);
 
+  const stackedLines = getSortedStackedLines(lines);
+
+  const regularLines = difference(lines, stackedLines);
+
+  const isLegendClicked = lines?.length <= 1;
+
+  const isDisplayedThreshold =
+    equals(resource?.type, ResourceType.anomalydetection) && !isLegendClicked;
+
+  const thresholdProps = {
+    getTime,
+    graphHeight,
+    leftScale,
+    regularLines,
+    rightScale,
+    secondUnit,
+    thirdUnit,
+    timeSeries,
+    xScale,
+  };
+
   return (
     <ClickAwayListener onClickAway={hideAddCommentTooltip}>
```

Comment:
Why would we want to hide things when in anomaly detection config mode?

Reply:
because since I use the same graph component, it contains other components for example user hovers over the graph and the comment is displayed to add it, for my case I no longer want these functionalities : 

https://user-images.githubusercontent.com/97687698/187468231-68c4bbf8-4e56-446f-b955-7622b5cf1651.mp4